test/helpers/framework.go

```diff
@@ -64,6 +64,8 @@ type LogStore interface {
 
 	GrepLogs(expr string, timeToWait time.Duration) (string, error)
 
+	RetrieveLogs() (map[string]string, error)
+
```

Comment:
Why do we need another log retrieval function on this interface? This blows up the interface. Consider using `ApplicationLogs` that returns a typed `logs` slice.

Reply:
Maybe a comment is in order here.  what does this provide that can not be fetched from one of the other methods?